game/src/ecs/entities/AnimationEntity.java

```diff
@@ -0,0 +1,39 @@
+package ecs.entities;
+
+import ecs.components.PositionComponent;
+import ecs.components.animation.AnimationComponent;
+import graphic.Animation;
+import java.util.logging.Logger;
+import starter.Game;
+import tools.Point;
+
+/** An entity used to display animations in the dungeon. */
+public class AnimationEntity extends Entity {
+
+    private static final Logger logger = Logger.getLogger(AnimationEntity.class.getName());
+
+    /**
+     * Creates a new entity animation. An entity used to display animations in the dungeon.
```

Comment:
Sollte der Hinweis mit dem Looping-Problem auch in die JavaDoc?

Reply:
Und der zweite Satz steht genauso im Klassen-JavaDoc.

Reply:
Ja habe das Looping problem mal mit aufgenommen und den reduntanten Satz rausgeschmissen.